src/index.html

```diff
@@ -1,114 +1,113 @@
+            <img class="water-icon" alt="An icon depicting a white drop of water in a blue circle" src="./images/water-icon.png">
+          </article>
+          <img class="man-icon" alt="An icon depicting a man tired from lifting weights" src="./images/little-man-icon.svg">
+          <article class="hydration__article--2 article--widget">
+            <h5>WEEKLY OZ</h5>
+            <canvas id="weekly-oz-graph"></canvas>
+            <ul id="weekly-oz"></ul>
+          </article>
+        </section>
       </section>
-    </section>
-    </section>
-    <section class="main__section--sleep main__sections">
-      <h3 class="section__h3--title">Sleep</h3>
-      <section class="sleep--container widget--container">
-        <article class="sleep__article--1 article--widget">
-          <h5>HOURS SLEPT</h5>
-          <img class="sleep-icon" src="./images/sleep-icon.png">
-          <div id="yesterday-sleep""></div>
-        </article>
-        <article class=" sleep__article--2 article--widget">
-            <h5>YOUR SLEEPING CHART</h5>
-            <canvas id="weekly-sleep"></canvas>
-        </article>
-        <article class="sleep__article--3 article--widget">
-          <h5>ALL-TIME STATS</h5>
-          <img class="trend-icon" src="./images/crown-icon.png">
-          <div id="all-time-sleep" class="all-time-sleep"></div>
-        </article>
       </section>
-    </section>
-    <section class="main__section--activity main__sections">
-      <h3 class="section__h3--title">Activity</h3>
-      <section class="activity--container--1 widget--container">
-        <article class="activity__article--1 article--widget">
-          <h5>TODAY'S ACTIVITY</h5>
-          <img class="exercise-icon" src="./images/exercise.icon.png">
-          <div id="daily-activity" class="article__widget--div"></div>
-        </article>
-        <img class="step-icon" src="./images/step-icon.png">
-        <article class="activity__article--2 article--widget">
-          <h5>WORLD COMPARISON</h5>
-          <img class="globe-icon" src="./images/globe-icon.png">
-          <div id="compare-activity"></div>
-        </article>
+      <section class="main__section--sleep main__sections">
+        <h3 class="section__h3--title">Sleep</h3>
+        <section class="sleep--container widget--container">
+          <article class="sleep__article--1 article--widget">
+            <h5>HOURS SLEPT</h5>
+            <img class="sleep-icon" alt="An icon depicting a yellow moon" src="./images/sleep-icon.png">
+            <div id="yesterday-sleep""></div>
+          </article>
+          <article class=" sleep__article--2 article--widget">
+              <h5>YOUR SLEEPING CHART</h5>
+              <canvas id="weekly-sleep"></canvas>
+          </article>
+          <article class="sleep__article--3 article--widget">
+            <h5>ALL-TIME STATS</h5>
+            <img class="trend-icon" alt="An icon depicting a yellow crown" src="./images/crown-icon.png">
```

Comment:
I'd be interested to see how alt tags are read, it may be that we don't need to start them with "an" depending on if the computer already does that.